index.js

```diff
@@ -323,6 +329,30 @@ const Instauto = async (db, browser, options) => {
     // Array.from(document.getElementsByTagName('a')).find(el => el.attributes?.href?.value.includes(`${username}/followers`)).innerText
   }
 
+ 
+  async function getCurrentFollowRequests() {
+    const url = `${instagramBaseUrl}/accounts/access_tool/current_follow_requests`;
+    await page.goto(url);
+    const viewMoreClass = '.L3NKy';
```

Comment:
I'm not sure if this is very safe to do? looks like some generated value that might change